examples/amp-story/player-story-attribution.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Story Player (Non-AMP)</title>
+    <script async src="../../dist/amp-story-player.js"></script>
+    <link href="../../build/css/amp-story-player-v0.css" rel='stylesheet' type='text/css'>
+    <style page>
+      body {
+        font-family: sans-serif;
+      }
+    </style>
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  </head>
+  <body>
+    <h1>This is a NON-AMP page that embeds a story below:</h1>
+
+    <amp-story-player style="width: 360px; height: 600px;">
+        <script type="application/json">
+          {
+            "appearance": {
```

Comment:
Does this need to be in a new "appearance" bucket? What are the other options that would go in there?

Reply:
So far we have these buckets
* `controls`: for the custom UI controls
* `behavior`: for fetching stories, circular wrapping, etc..

The `controls` bucket is the closest semantically but I don't think it feels right. It's not technically a "control" (like the volume, share, and close controls - which actually "control" something). 

The counter argument is that since it has an action (clicking it takes you to the publisher / creator site) it could be considered a control. 

I have no strong feelings here but I have a slight preference for the `appearance` bucket, I think it will be easier for publishers to understand and I can see other features fitting inside of it (e.g. footer). WDYT?

Reply:
I'll leave this up to you, I just wanted to make sure you considered both options : ))